vyper/compiler.py

```diff
@@ -51,4 +46,4 @@ def mk_full_signature(self, code, *args, **kwargs):
         return mk_full_signature(code, *args, **kwargs)
 
     def gas_estimate(self, code, *args, **kwargs):
```

Comment:
This method in the compiler class never gets tested? Why?

Reply:
I believe it's used outside our scope, I believe as part of the necessary methods of pyethereum Compiler class.